templates/minelev/varsel/hemmelig-nb.md

```diff
@@ -0,0 +1,31 @@
+---
+definition: notatmal
+language: nb
+watermark: {{#if preview}} true {{else}} false {{/if}}
+to: {{ school.name }}
+info:
+  sector: Seksjon for sektorutvikling
+  our-date: {{ isoDate created.timestamp }}
+  paragraph: Offl. § 13 jf. fvl. §13 (1)
+---
+
+# Varsel må sendes til {{ student.name }}
+
+Det er i dag produsert et varsel i MinElev til {{ student.name }} i klasse {{ student.classId }} fra {{ teacher.name }}.
+Siden eleven har hemmelig adresse skal ikke varselet sendes digitalt gjennom SvarUT. Varselet er arkivert i eleven sin elevmappe i 360.
+
+Administrasjonen på {{ school.name }} må sørge for å skrive ut varselet fra 360 for å gi det videre til {{ teacher.name }}. Det er {{ teacher.name }} som nå er ansvarlig å distribuere varselet videre til eleven. Er eleven under 18 år distribueres varselet etter de bestemmelsene som er gjort for eleven.
```

Comment:
Burdet det ikke vært en `for` i den midterste setningen?

Det er {{ teacher.name }} som nå er ansvarlig `for` å distribuere varselet videre til eleven.

Reply:
Fixed!